tests/test_drafts.py

```diff
@@ -65,3 +65,16 @@ def test_delete_draft(api_client):
     draft.save()
     # ... and delete it for real.
     draft.delete()
+
+
+@responses.activate
+@pytest.mark.usefixtures("mock_draft_saved_response")
+def test_draft_version(api_client):
+    draft = api_client.drafts.create()
+    assert 'version' not in draft
```

Comment:
@singingwolfboy With the API, the first time a draft is `POST`ed it will return a version of 0 (and no version needs to be passed in to this `POST`). Later, if a PUT request is made on that draft, you must pass `"version": 0`. The json response of the updated draft will include `"version": 1`.

Since it looks like the `create()` method only creates a local copy of the object and doesn't actually make an API request until `save()` is called it, then after the first `save()` the draft version returned should be 0 according to the logic above.

Could you change the other draft saves to be `update()` instead to be a little more explicit please, and make sure the draft version is 0 after the first save?